reference/docs-conceptual/dev-cross-plat/vscode/using-vscode.md

```diff
@@ -155,59 +155,61 @@ field. For example:
 
 You can add other PowerShell executable paths to the session menu through the
 [Visual Studio Code setting](https://code.visualstudio.com/docs/getstarted/settings):
-`powershell.powerShellAdditionalExePaths`.
+setting `powershell.powerShellAdditionalExePaths`.
```

Comment:
```suggestion
`powershell.powerShellAdditionalExePaths`.
```

Reply:
Good catch, my eyes went right over that other word.